src/scs/slsa-framework.md

```diff
@@ -0,0 +1,22 @@
+# SLSA Framework
+
+Supply chain Levels for Software Artifacts (SLSA) is a security framework for tampering prevention, integrity improvement and securing packages and infrastructure of a project.
+
+## Levels of assurance
+
+One of the requirements for the solution is to reach SLSA Level 4 and even go beyond that. This requires a lot of process changes as well as technical work. Current status for Thincompute project is visualized at the separate wiki page: Supply Chain Security 
+
+The SLSA model consists of 4 levels, offering incremental level of anti-tampering protection. Levels 1-3 represent milestones with certain integrity guarantees, whereas level 4 is an ideal end state.
+
+**Level 0** means no SLSA compliance and no guarantees are given.
+
+**Level 1** demands fully scripted/automated build process and provenance generation. It offers basic confidence of the origin of the software but doesn’t provide any tamper resistance.
+
+**Level 2** Introduces authenticated provenance generated by a hosted build at the same time demanding version control system usage. Provenance prevents tampering and offers trust in build service.
+
+**Level 3** offers auditability of the source and and the integrity of the provenance. Much stronger tampering protection is provided by preventing specific classes of threats such as cross-build contamination.
+
+**Level 4** requires two-peer source code review and a hermetic, reproducible build process. Overall Level 4 offers high degree of confidence in anti-tampering protection of the software.
+
+Important note is that SLSA level is not transitive, thus level of the artifact is not dependent on the level of dependencies, which are expected to have their own SLSA levels. This makes it possible to build Level 4 artifact from Level 0 dependencies. 
```

Comment:
Should we remove "Important note is that" and start with: SLSA level is not transitive...


...to build Level 4 artifact**s** from Level 0 dependencies. 
Or if you want to use a singular here, then: ...build **a** Level 4 artifact...